### PR TITLE
Translation of specification layout

### PR DESCRIPTION
**Pull request by shenlin192**
Hello @choudlet, my translation for the "Layout" in guidelines is finished. Please have a look.
If everything goes on well, translation for the "form" in patterns will be done in 3 weeks.

### CODE REVIEW

**Comment by afc163** [comment]
You should create a new `layout.zh-CN.md` for original file and remove chinese words from `layout.en-US.md`.

**Comment by shenlin192** [comment]
Ok, I will correct them.

File: docs/spec/layout.en-US.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+### Home page
+
+
+<img class="preview-img no-padding" align="right" src="https://zos.alipayobjects.com/rmsportal/olHkTiGQqfwThlgPIXzx.png">
+
+<img class="preview-img no-padding" align="right" src="https://zos.alipayobjects.com/rmsportal/uxbNrsFCmPFjYdhDowky.png">
+
+Home page is usually the first step for a user to understand a website or the products. Generally, home page consists of product drawings, brief product introductions, and user login interfaces. In the design, we recommend you to:
+
+- keep the copywriting clear and simply, which helps you better express the ideas.
```
Comment (afc163) [inline docs/spec/layout.en-US.md]: Keep

Comment (shenlin192) [inline docs/spec/layout.en-US.md]: Hello @afc163, do you mean you want me to capitalize all the sentences in list?

Comment (afc163) [inline docs/spec/layout.en-US.md]: Yes

File: docs/pattern/form.en-US.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+<br>
+
+<img class="preview-img" align="right" alt="左对齐" src="https://os.alipayobjects.com/rmsportal/eqUyDExbRlAQoas.png">
+
+左对齐。
+Align left
+
+- 优点：文字开头按阅读视线对齐，方便阅读；节约垂直空间。
+- 缺点：填写速度慢；标签长度和输入框弹性小。
+- 场景：希望用户放慢速度，仔细思考表单中的每个输入框。
```
Comment (afc163) [inline docs/pattern/form.en-US.md]: Do you plan to translate form in this PR too?

Comment (shenlin192) [inline docs/pattern/form.en-US.md]: Sorry. I made some mistakes. This PR is only for translation of Grid.

Comment (afc163) [inline docs/pattern/form.en-US.md]: Don't be. You are doing great job!

Comment (shenlin192) [inline docs/pattern/form.en-US.md]: My pleasure

**Pull request merged**